attestation-service/verifier/src/se/README.md

```diff
@@ -107,6 +107,7 @@ export SE_SKIP_CERTS_VERIFICATION=true
 ```
 DOCKER_BUILDKIT=1 docker build -t ghcr.io/confidential-containers/staged-images/kbs:latest --build-arg KBS_FEATURES=coco-as-builtin,openssl,resource,opa . -f kbs/docker/Dockerfile
 ```
+>Note: Please add `--debug` in statement like `cargo install` in file `kbs/docker/Dockerfile` if you're using a development host key document to skip HKD's signature verification.
```

Comment:
And the env var must be set

Reply:
@fitzthum for developer, I think we can just revise commands like `cargo install ...` in file `kbs/docker/Dockerfile` directly?